src/app/scripts/cac/control/cac-control-trip-options.js

```diff
@@ -322,13 +327,15 @@ CAC.Control.TripOptions = (function ($, Handlebars, moment, Modal, UserPreferenc
     function bikeModalOptions() {
 
         var source = [
-            '<li class="modal-list-indego">{{bikeShare}}</li>',
-            '<li class="modal-list-timing">{{timing}}</li>',
-            '<li class="modal-list-ride">{{bikeTriangle}}</li>'
+            '<li class="modal-list-indego {{#if setBikeShare}} selected{{/if}}">{{bikeShare}}</li>',
+            '<li class="modal-list-timing {{#if setTiming}} selected{{/if}}">{{timing}}</li>',
+            '<li class="modal-list-ride {{#if setTriangle}} selected{{/if}}">{{bikeTriangle}}</li>'
         ].join('');
 
         var bikeShare = options.defaultMenuText.bikeShare;
+        var setBikeShare = false;
```

Comment:
This chunk could be simplified to
```
var bikeShare = options.defaultMenuText.bikeShare;
var setBikeShare = !UserPreferences.isDefault('bikeShare');
if (setBikeShare) {
    if (UserPreferences.getPreference('bikeShare')) {
        bikeShare = 'Use Indego bike sharing';
    } else {
        bikeShare = 'Use my own bike';
    }
}
```
right?  Only two lines shorter, but it seems simpler to me.